### PR TITLE
CSVS-15

### PR DESCRIPTION
**Pull request by davidedelerma**

### CODE REVIEW

File: wwwroot/config.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
         // Enable experimental features, since its nice to have the bleeding edge.
         // Disable it if things go wrong with the experimental features or your users aren't robust to features which might change or disapear completly.
         "experimentalFeatures": true,
-
+        "cesiumIonAccessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI3ODY5M2VjZi01MGE5LTQwMGYtYmU0Ni1kOGIzNWUxYzZlNzQiLCJpZCI6Mjc1ODksInNjb3BlcyI6WyJhc3IiLCJnYyJdLCJpYXQiOjE1ODk3OTIyMjl9.P7XCpfHLYOnfxZqq89HQrLEMbFe6XzVKwBAjrH68GLs",
```
Comment (Juanezm) [inline wwwroot/config.json:63]: Shouldn't we add this using env vars?

Comment (davidedelerma) [reply wwwroot/config.json:63]: The config file is overwritten in kubernetes with good envs this is just a trial one. In therou we won't use cesium and 3D stuffs as it requires a paid licence :(

Comment (davidedelerma) [reply wwwroot/config.json:63]: So we would use just the 2d bit

Comment (davidedelerma) [reply wwwroot/config.json:63]: https://cesium.com/pricing/

**Pull request merged**